nakhll_market/serializers.py

```diff
@@ -684,7 +684,8 @@ class Meta:
         model = Shop
         fields = [
             'Title', 'Slug', 'Image', 'image_thumbnail_url',
-            'bank_account', 'social_media', 'Description', 'FK_ShopManager',
+            'bank_account', 'social_media', 'Description', 'FK_ShopManager',"state","BigCity",
+            "City","Location"
```

Comment:
please, `"` OR `'` NOT BOTH.
```suggestion
            'bank_account', 'social_media', 'Description', 'FK_ShopManager', 'state', 'BigCity',
            'City', 'Location'